src/js/parsers/post.js

```diff
@@ -52,7 +52,7 @@ export default {
 
     // walk up from the textNode until the rootNode, converting each
     // parentNode into a markup
-    function collectMarkups(textNode, rootNode) {
+    const  collectMarkups = (textNode, rootNode) =>{
```

Comment:
FWIW I find this pattern (creating functions inside a function scope) to be very un-friendly to read, and have foot-gun-esq problems of scope (what variables are leaking in etc). Unless we're generating callbacks, I'm not crazy about using locally scoped function for helpers.